### PR TITLE
[Merged by Bors] - chore: process and remove adaptation_note for nightly-2024-09-06

### PR DESCRIPTION
**Pull request by jcommelin**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Int/Defs.lean [hunk 1]
```diff
@@ -43,12 +43,7 @@ protected lemma le_antisymm_iff : a = b ↔ a ≤ b ∧ b ≤ a :=
 protected lemma le_iff_eq_or_lt : a ≤ b ↔ a = b ∨ a < b := by
   rw [Int.le_antisymm_iff, Int.lt_iff_le_not_le, ← and_or_left]; simp [em]
 
-#adaptation_note
-/--
-After nightly-2024-09-06 we can remove the `_root_` prefix below.
--/
-protected lemma le_iff_lt_or_eq : a ≤ b ↔ a < b ∨ a = b := by rw [Int.le_iff_eq_or_lt,
-  _root_.or_comm]
```
Comment (YaelDillies) [inline Mathlib/Data/Int/Defs.lean:51]: How is this even valid indenting?!

File: Mathlib/Data/List/Chain.lean [hunk 2]
```diff
@@ -47,10 +47,6 @@ theorem Chain.iff_mem {a : α} {l : List α} :
 theorem chain_singleton {a b : α} : Chain R a [b] ↔ R a b := by
   simp only [chain_cons, Chain.nil, and_true]
 
-#adaptation_note
-/--
-After nightly-2024-09-06 we can remove the `_root_` prefix below.
--/
```
Comment (YaelDillies) [inline Mathlib/Data/List/Chain.lean:53]: Looks like this note was adapting nothing?

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by j-loreaux** [comment]
bors d+

**Comment by mattrobball** [comment]
bors delegate+

**Pull request closed without merging**